easybuild/easyconfigs/n/netCDF/netCDF-4.3.1.1-ictce-6.1.5.eb

```diff
@@ -0,0 +1,29 @@
+name = 'netCDF'
+version = '4.3.1.1'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'ictce', 'version': '6.1.5'}
+toolchainopts = {'opt': True, 'pic': True, 'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.unidata.ucar.edu/downloads/netcdf/ftp/']
+
+dependencies = [('HDF5', '1.8.12', '-zlib-1.2.8')]
+
+builddependencies = [
+    ('cURL', '7.34.0'),
```

Comment:
`cURL` should be a runtime dep (based on existing easyconfigs)

missing build deps: `Autotools`, `CMake`, `Doxygen` (see existing `netCDF` easyconfigs)